docs/source-2.0/guides/smithy-cli/cli_installation.rst

```diff
@@ -0,0 +1,470 @@
+        to install the CLI in the default location (``/usr/local/smithy``).
+        Alternatively, you may set your own install path, which should
+        mitigate permissions issues when not running with ``sudo``.
+
+
+============
+Installation
+============
+
+The Smithy CLI is supported on the following platforms:
+
+- MacOS (x86, ARM)
+- Linux (x86, ARM)
+- Windows (x64)
+
+Releases of the Smithy CLI can be found on the `Smithy GitHub releases`_ page.
+
+.. tab:: MacOS
+
+    .. tab:: Homebrew
+
+        You can get the CLI via Homebrew by first tapping into the official
+        `AWS Homebrew Tap`_, and then installing the ``smithy-cli`` formula.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ brew tap aws/tap && brew install smithy-cli
+
+        After the install completes, you may run ``smithy --help`` to verify
+        your installation.
+
+    .. tab:: Manual (x86)
+
+        First, retrieve the latest smithy installation from the
+        `Smithy GitHub releases`_.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ mkdir -p smithy-install/smithy &&
+                curl -L |release-uri|/|mac-tar| -o smithy-install/|mac-tar| &&
+                tar xvzf smithy-install/|mac-tar| -C smithy-install/smithy
+
+        .. seealso::
+            |verification-note|
+
+        Now, run the installer (``./install``) located in the newly created
+        directory (``smithy-install/smithy``).
+
+        .. important::
+            |sudo-note|
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ sudo smithy-install/smithy/install
+
+        .. note::
+            |install-note|
+
+        After the install completes, you may run ``smithy --help`` to verify
+        your installation. You may also delete all the files downloaded for
+        performing the installation (under ``smithy-install/``)
+
+    .. tab:: Manual (ARM)
+
+        First, retrieve the latest smithy installation from the
+        `Smithy GitHub releases`_.
+
+        .. code-block:: sh
+            :caption: /bin/sh
+            :substitutions:
+
+            $ mkdir -p smithy-install/smithy &&
+                curl -L |release-uri|/|mac-arm-tar| -o smithy-install/|mac-arm-tar| &&
+                tar xvzf smithy-install/|mac-arm-tar| -C smithy-install/smithy
+
+        .. seealso::
+            |verification-note|
+
+        Now, run the installer (``./install``) located in the newly created
+        directory (``smithy-install/smithy``).
+
+        .. important::
+            |sudo-note|
+
+        .. code-block:: sh
+            :caption: /bin/sh
+
+            $ sudo smithy-install/smithy/install
+
+        .. note::
+            |install-note|
+
+        After the install completes, you may run ``smithy --help`` to verify
+        your installation. You may also delete all the files downloaded for
```

Comment:
Why not just add the correct rm command to the above codeblock to remove the smithy-install/ dir?